src/xenia/cpu/backend/x64/x64_seq_vector.cc

```diff
@@ -708,6 +708,32 @@ struct VECTOR_SHL_V128
 
   static void EmitInt8(X64Emitter& e, const EmitArgType& i) {
     // TODO(benvanik): native version (with shift magic).
+
+    if (i.src2.is_constant) {
+      unsigned char n = i.src2.constant().u8[0];
+      bool all = true;
+      for (unsigned j = 1; j < 16; ++j) {
+        if (i.src2.constant().u8[j] != n) {
+          all = false;
+          break;
+        }
+      }
+      if (all) {
+        // we build a mask to eliminate the bits that will be shifted out of the
+        // bytes so that they do not enter the adjacent bytes
+        vec128_t shiftout_mask; 
```

Comment:
I feel like this would be better declared after the `mask` declaration -- closer to its initialization.